routes/personal/schema.js

```diff
@@ -1,56 +1,74 @@
 /* istanbul ignore file */
 
-const isValidDate = require('../../utils/').isValidDate
+const isValidDate = require('../../utils/custom/validate.helpers.js').isValidDate
+const validateTelephoneFormat = require('../../utils/custom/validate.helpers.js').validateTelephoneFormat
+const validateDateFormat = require('../../utils/custom/validate.helpers.js').validateDateFormat
 
 const Schema = {
-  fullname: {
+  social: {
     isLength: {
-      errorMessage: 'errors.fullname.length',
-      options: { min: 3, max: 200 },
+      errorMessage: 'Social seems to be incorrect format',
+      options: { min: 11, max: 11 },
     },
   },
-  email: {
+  first_name: {
     isLength: {
-      errorMessage: 'errors.email.length',
-      options: { min: 3, max: 200 },
+      errorMessage: 'First name is required',
+      options: { min: 3, max: 40 },
     },
   },
-  expiry: {
+  last_name: {
+    isLength: {
+      errorMessage: 'Last name is required',
+      options: { min: 3, max: 40 },
```

Comment:
Not we should put in minimums for last names since a fairly common last name is ng in some parts of the world, and so they would not be able to enter the data.

Reply:
Also validating names is hard

Reply:
agreed. I'm going to set length to min:1 just to make the fields required.